test/tests/performance/pbs_jobperf.py

```diff
@@ -95,6 +95,19 @@ def wait_for_job_finish(self, jobiter=True):
         self.server.expect(JOB, {'job_state=F': total_jobs}, extend='x',
                            interval=20, trigger_sched_cycle=False)
 
+    def delete_jobs_per_user(self, users, num_users):
+        """
+        Delete jobs faster by providing more job id's at once
+        """
+        bin_path = str(os.path.join(
+            self.server.pbs_conf['PBS_EXEC'], 'bin'))
+        qdel = str(os.path.join(bin_path, 'qdel'))
+        qdel = qdel + ' -W force'
+        for u in range(0, num_users):
```

Comment:
Does this mean that this test suite cannot be run as a non-root user? because I'm not sure you will be able to delete jobs from other users

Reply:
Test can be run as non root user . This will work fine as in PTL current user will be added as manager he should be able to do qselect and qdel on other users jobs as well.